tests/common/devices.py

```diff
@@ -227,22 +232,26 @@ def get_pmon_daemon_list(self):
         logging.info("Pmon daemon list for this platform is %s" % str(daemon_list))
         return daemon_list
 
-    def get_up_time(self):
-        up_time_text = self.command("uptime -s")["stdout"]
-        return datetime.strptime(up_time_text, "%Y-%m-%d %H:%M:%S")
+class FanoutHost(Localhost):
+    def __init__(self, ansible_adhoc, localhost, duthost):
+        Localhost.__init__(self, ansible_adhoc)
+        self.ansible_playbook = os.path.realpath(os.path.join(os.path.dirname(__file__), "../scripts/exec_template.yml"))
+        self.playbook_template = 'cd {ansible_path}; ansible-playbook {playbook} -i lab -l {fanout_host} --extra-vars \'{extra_vars}\' -vvvvv'
+        self.localhost = localhost
+        self.duthost = duthost
+        self.fanout_host = None
+        self.facts = {}
+        self.gather_facts()
 
-    def get_now_time(self):
-        now_time_text = self.command('date +"%Y-%m-%d %H:%M:%S"')["stdout"]
-        return datetime.strptime(now_time_text, "%Y-%m-%d %H:%M:%S")
+    def exec_template(self, **kwargs):
+        cli_cmd = self.playbook_template.format(ansible_path=ANSIBLE_ROOT, playbook=self.ansible_playbook,
+                                                fanout_host=self.fanout_host, extra_vars=json.dumps(kwargs))
+        res = self.localhost.shell(cli_cmd)
 
-    def get_uptime(self):
-        return self.get_now_time() - self.get_up_time()
+        if res["rc"] != 0:
+            raise Exception("Unable to execute template\n{}".format(res["stdout"]))
 
-    def get_networking_uptime(self):
-        start_time = self.get_service_props("networking", props=["ExecMainStartTimestamp",])
-        try:
-            return self.get_now_time() - datetime.strptime(start_time["ExecMainStartTimestamp"],
-                                                           "%a %Y-%m-%d %H:%M:%S UTC")
-        except Exception as e:
-            self.logger.error("Exception raised while getting networking restart time: %s" % repr(e))
-            return None
```

Comment:
These uptime related methods are required by the new sanity check. Why remove them?

Reply:
Please clarify what do you mean, what is removed? Maybe here was not the latest changes from master..?

Reply:
I mean some methods already in devices.py are removed in this PR. For example "get_uptime". In this version, there is no such issue.

Reply:
Those methods which you listed are present in "devices.py".
Also when we will have stable develop, I will merge my changes with the latest changes from master.